library/Zend/Controller/Action.php

```diff
@@ -91,6 +91,12 @@ abstract class Zend_Controller_Action implements Zend_Controller_Action_Interfac
      */
     public $view;
 
+    /**
+     * compataibility for php 8.2 to stop error Deprecated: Creation of dynamic property 
+     * @var object 
+     */
+    public  $contexts = null; 
```

Comment:
why not use a typed property?

Reply:
Because it PHP.

Reply:
The PHPDoc `@var object` now suggests it is an `object`, which it is not, it is an `array`